src/igor/sync.go

```diff
@@ -13,27 +13,54 @@ var cmdSync = &Command{
 	UsageLine: "sync",
 	Short:     "synchronize igor data",
 	Long: `
-Does an internal check to verify the integrity of the data file.
+Does an internal check to verify the integrity of the data file. Can report and attempt to clean.
 
-OPTIONAL FLAGS:
+SYNOPSIS
+	igor sync <[-d] [-f]> [-q]
 
-	-v	Verbose - Prints additioanl information
+OPTIONS
+
+	-f, -force
+	    Will force sync to fix inconsistencies found in addition to reporting
+
+	-d, -dry_run
+	    Does not attempt to make any corrections, only reports inconsistencies
+
+	-q, -quiet
+	    Suppress reports, only report errors
 	`,
 }
 
-var subL bool // -l
-var subV bool // -v
+var subF bool   // -f
+var force bool  // -force
+var subD bool   // -d
+var dryRun bool // -dry-run
+var subQ bool   // -q
+var quiet bool  // -quiet
 
 func init() {
 	// break init cycle
 	cmdSync.Run = runSync
 
-	cmdSync.Flag.BoolVar(&subL, "l", false, "")
-	cmdSync.Flag.BoolVar(&subV, "v", false, "")
+	cmdSync.Flag.BoolVar(&subF, "f", false, "")
+	cmdSync.Flag.BoolVar(&force, "force", false, "")
+	cmdSync.Flag.BoolVar(&subD, "d", false, "")
+	cmdSync.Flag.BoolVar(&dryRun, "dry_run", false, "")
```

Comment:
Does this map the -dry-run? Has an underscore instead of hyphen.

Reply:
Weirdly, the hyphen was not working before, so I switched to underscore. But now it's working, so my problem must have been elsewhere. I'll switch back.